posts/2024-05-22_gno-studio-intro/README.md

```diff
@@ -0,0 +1,123 @@
+---
+publication_date: 2024-05-27T13:37:00Z
+slug: gno-studio-intro
+tags: [gno-studio, dev, devx, connect, gno.land]
```

Comment:
Can we change these with the following ones?
<img width="629" alt="image" src="https://github.com/gnolang/blog/assets/6023117/e2628907-4c7e-42f6-a614-175be7caf382">

Reply:
[d74b73d](https://github.com/gnolang/blog/pull/84/commits/d74b73dc494c183a79210e5b377575643b4c2ecc)